learnbot_dsl/blocksConfig/basic/Operators.conf

```diff
@@ -44,14 +44,14 @@
     "category" : "operator",
     "name" : "=",
     "shape" : ["block3"],
-    "tooltip" : {"ES": "Iguala el bloque de la izquierda al bloque de la derecha", "EN": ""}
+    "tooltip" : {"ES": "Iguala el bloque de la izquierda al bloque de la derecha", "EN": "Equalizes the left block to the right block."}
 },
 {
     "type" : "operator",
     "category" : "operator",
     "name" : "==",
     "shape" : ["block3"],
-    "tooltip" : {"ES": "Compara los bloques conectados retornando verdadero en caso de que sean iguales y falso en caso contrario", "EN": ""},
+    "tooltip" : {"ES": "Compara los bloques conectados retornando verdadero en caso de que sean iguales y falso en caso contrario", "EN": "Compares the connected blocks, returning true if they are equal and false if not."},
```

Comment:
```suggestion
    "tooltip" : {"ES": "Compara los bloques conectados retornando verdadero en caso de que sean iguales y falso en caso contrario", "EN": "Compares the connected blocks, returning true if they are equal and false otherwise."},
```